tests/unit/test_endpoint_provider.py

```diff
@@ -411,6 +414,14 @@ def test_alerts_in_both_formats_are_recursively_aggregated(self):
 
         self.assertEqual({}, DeepDiff(expected_rules_file, rules_file_dict, ignore_order=True))
 
+    def test_unit_not_in_alert_labels(self):
+        rules = AlertRules(topology=self.topology)
+        rules.add_path(os.path.join(self.sandbox.root, "rules", "prom"), recursive=True)
+        rules_file_dict = rules.as_dict()
+        for group in rules_file_dict["groups"]:
+            for rule in group["rules"]:
+                self.assertTrue("juju_unit" not in rule["labels"])
```

Comment:
Is this different from `test_each_alert_rule_is_topology_labeled`?
Should this also have the `expr` check?

Reply:
This test is probably redundant and was added in @dstathis commit. @dstathis is this test still needed.

Reply:
Yes we still need this. Your test checks the opposite case.